src/vm/i386/cgenx86.cpp

```diff
@@ -402,6 +402,11 @@ void HelperMethodFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 
     ClearRegDisplayArgumentAndScratchRegisters(pRD);
 
+    //
+    // Fix up ResumeSp
+    //
+    pRD->pCurrentContext->ResumeEsp = EECodeManager::GetResumeSp(pRD);
```

Comment:
It is possible, but does not matter as we use ResumeEsp only when resuming from managed code.